### ISSUE TITLE
Nightly K8s Matrix CI 2023-07-25: Failed

### ISSUE THREAD

**Issue by weizhoublue** [spidernet-io/spiderpool#2075]
action url: https://github.com/spidernet-io/spiderpool/actions/runs/5661331216

**Comment by ty-dc** [comment]
The reasons for failure in ci are as follows:
1. The ipv6 abbreviation problem is similar to the #2070 problem.
2. Randomize to overlapping IP addresses, will be fixed in #2073
3. Deleted multus cr but didn't rebuild automatically?

**Comment by Icarus9913** [comment]
```text
{"level":"INFO","ts":"2023-07-25T20:38:28.058Z","logger":"MultusConfig-Informer","caller":"multuscniconfig/multusconfig_informer.go:337","msg":"try to create net-attach-def &{{ } {test-multus-1a20fec71f  ns-98242f597f    0 0001-01-01 00:00:00 +0000 UTC <nil> <nil> map[] map[] [{spiderpool.spidernet.io/v2beta1 SpiderMultusConfig test-multus-1a20fec71f 2cb8ffd1-4ac3-448b-939e-8be980f5c046 0xc001691f99 0xc001691f98}] [] []} {{\"cniVersion\":\"0.3.1\",\"name\":\"test-multus-1a20fec71f\",\"plugins\":[{\"type\":\"macvlan\",\"ipam\":{\"type\":\"spiderpool\"},\"master\":\"eth0\",\"mode\":\"bridge\"},{\"type\":\"coordinator\",\"ipam\":{},\"dns\":{},\"detectGateway\":false,\"tunePodRoutes\":true,\"mode\":\"disabled\",\"hostRuleTable\":500,\"detectIPConflict\":false}]}}} for MultusConfg ns-98242f597f/test-multus-1a20fec71f"}
```

If the Multus net-attach-def was deleted, the SpiderMultusConfig informer would recreate it after 1 minute. So, the e2e case need to improve the check duration.
@ty-dc
